ksql-engine/src/main/java/io/confluent/ksql/execution/json/ParsedTypesPlanJsonMapper.java

```diff
@@ -15,16 +15,13 @@
 
 package io.confluent.ksql.execution.json;
 
-import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
 import io.confluent.ksql.json.KsqlTypesSerializationModule;
 import io.confluent.ksql.parser.json.KsqlParserSerializationModule;
 import io.confluent.ksql.parser.json.KsqlTypesDeserializationModule;
 
-public final class PlanJsonMapper {
-  private PlanJsonMapper() {
+public final class ParsedTypesPlanJsonMapper {
```

Comment:
Rather than splitting this, why not just move the testing into the `ksql-functional-testing` module, which has access to both?